sklearn/svm/tests/test_svm.py

```diff
@@ -852,3 +854,13 @@ def test_hasattr_predict_proba():
     assert_true(hasattr(G, 'predict_proba'))
     msg = "predict_proba is not available when fitted with probability=False"
     assert_raise_message(NotFittedError, msg, G.predict_proba, iris.data)
+
+
+def test_decision_function_shape_svc():
```

Comment:
I thought the issue was unrelated to OneVsRestClassifier and only related to there being two classes in SVC.